src/app/layout.jsx

```diff
@@ -0,0 +1,32 @@
+import { Providers } from './providers';
+
+export default function RootLayout({children}) {
+  return (
+    <html lang="en">
```

Comment:
Cambiar el lenguaje a español

```suggestion
    <html lang="es">
```